.github/workflows/run-tests.yml

```diff
@@ -28,4 +28,4 @@ jobs:
     uses: ./.github/workflows/tests.yml
     with:
       ref: ${{ github.ref }}
-      nox-image: "docker.fluence.dev/nox:renovate-avm_4905_1"
+      nox-image: "docker.fluence.dev/nox:feat-vm-425-aquavm-mem-limits-from-config-2_5056_1"
```

Comment:
should the specific image still be used? It's good if latest release is used or something

Reply:
Yeap, specific image is necessary

Reply:
Otherwise tests won't pass